hmri_create_b1map.m

```diff
@@ -220,6 +220,28 @@
 P    = b1map_params.b1input; % B1 data - 11 pairs
 Q    = b1map_params.b0input; % B0 data - 3 volumes
 
+% splitting images in echo1 and echo2 volumes
+% wold be nice to check that echo times are as expected

```

Comment:
Suggestion: rephrase as
"% TODO: check that echo times are as expected"

Reply:
Done